sycl/include/sycl/ext/oneapi/bindless_images.hpp

```diff
@@ -1217,6 +1267,23 @@ void write_image_array(const unsampled_image_handle &imageHandle
 #endif
 }
 
+/**
+ *  @brief   Write to an unsampled cubemap using its handle
+ *
+ *  @tparam  DataT The data type to write
+ *
+ *  @param   imageHandle The image handle
+ *  @param   coords The coordinates at which to write image data (int2 only)
+ *  @param   face The cubemap face at which to write
+ *  @param   color The data to write
+ */
+template <typename DataT>
+void write_cubemap(const unsampled_image_handle &imageHandle,
```

Comment:
`imageHandle` is not const in the specification. Considering that `imageHandle` is indeed const in both `fetch_cubemap` and `sample_cubemap`, maybe the error is in the spec rather than here, right?

Reply:
Good catch. There's quite a bit of inconsistency with how we're applying `const` to these handles across the spec and implementation. When it comes to writing, the handles should not be `const`. I've applied a fix to the cubemaps funcs and elsewhere in the spec and impl where necessary.

Reply:
I'm good with it, as long as implementation matches spec.